controllers/provisioning_controller.go

```diff
@@ -250,6 +250,10 @@ func (r *ProvisioningReconciler) Reconcile(ctx context.Context, req ctrl.Request
 			"unable to put %q ClusterOperator in Available state", clusterOperatorName)
 	}
 
+	if _, ok := baremetalConfig.Annotations["provisioning.metal3.io/paused"]; ok {
+		return ctrl.Result{}, fmt.Errorf("provisioning CR paused")
```

Comment:
Maybe it's a case for RequeueAfter rather than an error? Especially since we don't want an immediate requeue.

Reply:
Done